device-types/Cisco/Meraki-MR56.yaml

```diff
@@ -2,7 +2,6 @@
 manufacturer: Cisco
 model: MR56
 slug: MR56
-u_height: 0
```

Comment:
These are not rack mountable to my knowledge. We should leave `u_height: 0` defined